srcpkgs/aide/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'aide'
+pkgname=aide
+version=0.16.2
+revision=1
+build_style=gnu-configure
+configure_args="--with-mhash --with-posix-acl --with-prelink --with-xattr
+ --with-zlib --with-e2fsattrs --disable-static"
+conf_files="/etc/aide.conf"
+make_dirs="/var/lib/aide 0750 root root /var/log/aide 0750 root root"
```

Comment:
There should be a newline separating the two make_dirs entries.

> By convention, there is only one entry of `dir perms user group` per line.

(Snippet from the make_dirs paragraph of Manual.md)

Reply:
Ah, I misread that line.